pkg/cache/clusterqueue_snapshot.go

```diff
@@ -221,3 +223,13 @@ func (c *ClusterQueueSnapshot) FindTopologyAssignmentsForWorkload(
 func (c *ClusterQueueSnapshot) IsTASOnly() bool {
 	return c.tasOnly
 }
+
+func (c *ClusterQueueSnapshot) PathToRoot() iter.Seq[*CohortSnapshot] {
```

Comment:
please document these functions, that they return all Cohorts, except for the root